addons/clones/userscript.js

```diff
@@ -35,14 +37,27 @@ export default async function ({ addon, global, console, msg }) {
 
   const cache = Array(301)
     .fill()
-    .map((_, i) => msg("clones", { cloneCount: i }));
+    //loads the blank message in if shownotext setting is enabled
+    .map((_, i) => {
+      if (showIconOnly) {
+        msg("clones", { cloneCount: i })
+      } else {
+        i
+      }
+    });
 
   function doCloneChecks() {
     const v = vm.runtime._cloneCounter;
     // performance
     if (v === lastChecked) return;
     countContainerContainer.dataset.count = lastChecked = v;
-    count.dataset.str = cache[v] || msg("clones", { cloneCount: v });
+    if(showIconOnly){
+      //it caches the blank message if shownotext setting is enabled
+      count.dataset.str = cache[v] || v;
+    }
+    else {
+      count.dataset.str = cache[v] || msg("blank", { cloneCount: v });
```

Comment:
I think you accidentally used the balnk message instead of the clone one here. 
```suggestion
      count.dataset.str = cache[v] || msg("clones", { cloneCount: v });
```